src/gf.c

```diff
@@ -11,13 +11,11 @@
 #include <stdlib.h>
 #include <string.h>
 #include <assert.h>
-#ifdef _OS_WINDOWS_
-#include <malloc.h>
```

Comment:
Does this also mean that we've been `include`ing `<unistd.h>` and it was fine on windows? (maybe not MSVC).